### ISSUE TITLE
[TF2] Add ConVar to adjust the hard movement speed cap

### ISSUE THREAD

**Issue by Marioiscool246** [ValveSoftware/Source-1-Games#4572]
Anyone who has been tinkering with TF2 such as modifying the class move speed via attributes or the class script file most likely knows about the 520 HU/s hard speed cap for player movement. It would be nice to have a replicated server console variable to set this speed cap for those who would want to adjust it.

**Comment by FlaminSarge** [comment]
(This is currently implemented as a serverside modification e.g. https://github.com/FlaminSarge/tf_maxspeed_patch but it would definitely be way nicer as a replicated cvar so that client prediction issues can be mitigated).

**Comment by Marioiscool246** [comment]
> (This is currently implemented as a serverside modification e.g. https://github.com/FlaminSarge/tf_maxspeed_patch but it would definitely be way nicer as a replicated cvar so that client prediction issues can be mitigated).

Yeah, I wrote a server mod that adjusts this directly back when I was hosting a sv_cheats 1 server but the prediction errors at high latencies are just too jarring.
